.changeset/gold-snails-dream.md

```diff
@@ -0,0 +1,7 @@
+---
+"@comet/admin": minor
+---
+
+Add a new `FillSpace` component to replace `ToolbarFillSpace` and `AppHeaderFillSpace`
+
+`ToolbarFillSpace` and `AppHeaderFillSpace` are now deprecated and will be removed in the next major version.
```

Comment:
We usually don't remove deprecated code in the next major version, but when maintaining the code becomes tedious. Is there a specific reason why you'd want to remove it asap?

Reply:
No good reason just thought it makes sense to remove it asap. I updated the changeset. 

> We usually don't remove deprecated code in the next major version, but when maintaining the code becomes tedious

Makes sense, didn't know that 👍🏼